addons/event/models/event_event.py

```diff
@@ -88,8 +88,7 @@ class EventEvent(models.Model):
     _order = 'date_begin'
 
     def _get_default_stage_id(self):
-        event_stages = self.env['event.stage'].search([])
-        return event_stages[0] if event_stages else False
+        return self.env['event.stage'].search([], limit=1)
```

Comment:
I'm guessing returning an empty browse record instead of False previously does not impact the default behavior?

Reply:
Everything is working well, even without stages, indeed.